pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -585,6 +586,7 @@ private BrokerResponseNative handleRequest(long requestId, String query, JsonNod
     brokerResponse.setTimeUsedMs(totalTimeMs);
     requestContext.setQueryProcessingTime(totalTimeMs);
     augmentStatistics(requestContext, brokerResponse);
+    _brokerMetrics.setValueOfTableGauge(rawTableName, BrokerGauge.QUERY_TOTAL_TIME_MS, totalTimeMs);
```

Comment:
should this be a BrokerQueryPhase (in addition to Gauge) so we can get the percentiles, average etc?

Reply:
yeah. I wasn't sure if this falls under the "query phase" metrics as it seemed to be applicable for each stage.  I don't think percentiles/average etc are related to `BrokerQueryPhase`, but rather to the metric type itself ? 
Please correct me if I have misunderstood.  :)

Reply:
To confirm - the new metric will include the total  latency observed at broker right from the time of arrival and all the other phases ?

Reply:
@siddharthteotia yes.. that's right. 
@npawar I think we need to use a `Timer` metric type if you need percentiles and average. I can change to use that. `BrokerQueryPhase` just labels the phase name to the metric name. Under the hood, it is a timer metric.

Reply:
`Timer` has rates and distribution stats. `Gauges` don't. (https://metrics.dropwizard.io/4.2.0/getting-started.html#timers)[https://metrics.dropwizard.io/4.2.0/getting-started.html#timers]

I also verified the metrics locally - it is seen in mbeans and getting published to prometheus.